py/selenium/webdriver/chrome/options.py

```diff
@@ -25,5 +26,9 @@ class Options(ChromiumOptions):
     def default_capabilities(self) -> dict:
         return DesiredCapabilities.CHROME.copy()
 
-    def enable_mobile(self, android_package="com.android.chrome", android_activity=None, device_serial=None):
+    def enable_mobile(self,
+                      android_package: str = "com.android.chrome",
+                      android_activity: Optional[str] = None,
+                      device_serial: Optional[str] = None
+                      ) -> NoReturn:
```

Comment:
This needs to be `None` not `NoReturn`

Reply:
I corrected that and pushed it.  Should we be revisiting `NoReturn` usage in other places such as https://github.com/SeleniumHQ/selenium/blob/trunk/py/selenium/webdriver/firefox/webdriver.py#L186 ?

Reply:
Yes! @colons already did that here — #10202 
I was trying to figure out why the `bazel buildd //py:test-remote` is failing before merging that one, but as discussed on Slack, I can't seem to get it to run on my machine at all.